lib/archethic/db/cassandra_impl.ex

```diff
@@ -128,7 +128,7 @@ defmodule ArchEthic.DB.CassandraImpl do
          address,
          after_time
        ) do
-    {" SELECT transaction_address FROM archethic.transaction_chains WHERE chain_address = ? AND transaction_timestamp >=  ? ",
+    {" SELECT transaction_address FROM archethic.transaction_chains WHERE chain_address = ? AND transaction_timestamp < ? ",
```

Comment:
Hi, please <= ,
`{" SELECT transaction_address FROM archethic.transaction_chains WHERE chain_address = ? AND transaction_timestamp <= ? ",` It fixes docker testnet issues.

Reply:
the previous state was: 
https://github.com/archethic-foundation/archethic-node/blob/c7b006a94e3621347161bfaee652fd7b9d8b4d6c/lib/archethic/p2p/message.ex#L1003

So I don't want to introduce something not testing properly